docs/src/clusters.md

```diff
@@ -147,7 +147,11 @@ $ solana-validator \
     --rpc-port 8899 \
     --private-rpc \
     --dynamic-port-range 8000-8010 \
-    --entrypoint mainnet-beta.solana.com:8001 \
```

Comment:
`mainnet-beta.solana.com` still exists, I'm just de-documenting it for consistency